artifactory/commands/golang/publish.go

```diff
@@ -197,7 +197,7 @@ func archive(moduleName, version, projectPath, tempDir string) (name string, zip
 			}
 		}
 	}()
-	err = archiveProject(tempFile, projectPath, moduleName, version)
+	err = archiveProject(tempFile, projectPath, moduleName, version, excludedPatterns)
 	if err != nil {
```

Comment:
Let's reduce the above 2 lines into 1 line.

Reply:
Done